src/main/resources/resources.properties

```diff
@@ -56,64 +56,64 @@ presentation.dependencyStatus.depError=Error in dependencies
-ui.toolWindow.subTab.exercises.submission.selectModule=Please select from which module you would like to submit:
+ui.toolWindow.subTab.exercises.submission.submitExercise=Submit Assignment
+ui.toolWindow.subTab.exercises.submission.selectModule=Please select which module you wish to submit:
 ui.toolWindow.subTab.exercises.submission.selectModuleShort=Select Module
 ui.toolWindow.subTab.exercises.submission.selectAModule=Select a module
 
 ## CourseProject
-ui.courseProject.dialogs.showRestartDialog.title=Restart IntelliJ IDEA
+ui.courseProject.dialogs.showRestartDialog.title=Restart IntelliJ
 ui.courseProject.dialogs.showRestartDialog.message=IntelliJ IDEA will now restart to reload settings.
 ui.courseProject.dialogs.showRestartDialog.okText=OK
 ui.courseProject.dialogs.showRestartDialog.cancelText=Cancel
 ui.courseProject.dialogs.showErrorDialog.networkError.title=Network Error
-ui.courseProject.dialogs.showErrorDialog.networkError.message=An error occurred while creating a course project. Please check your network connection and try again, or contact the course staff if the issue persists.
+ui.courseProject.dialogs.showErrorDialog.networkError.message=An error occurred while creating the course project. Please check your network connection and try again. If the issue persists, contact the course staff.
 ui.courseProject.dialogs.showErrorDialog.malformedCourseConfiguration.title=Course Configuration Error
-ui.courseProject.dialogs.showErrorDialog.malformedCourseConfiguration.message=An error occurred while reading the course configuration. Please contact the course staff.
+ui.courseProject.dialogs.showErrorDialog.malformedCourseConfiguration.message=The A+ Courses plugin could not read the course configuration, perhaps due to a mistake by the course staff.\nIf the problem persists, please contact the staff.
 
-ui.courseProject.form.textTemplate=<html><body>The currently opened project will be turned into a project for <b>{0}</b>.</body><html>
-ui.courseProject.form.infoText=<html><body>The A+ Courses plugin will adjust IntelliJ IDEA settings. This helps use IDEA for coursework.</body></html>
+ui.courseProject.form.textTemplate=<html><body>This project will be turned into a project for <b>{0}</b>.</body><html>
+ui.courseProject.form.infoText=<html><body>The A+ Courses plugin will now adjust IntelliJ''s settings. This helps you use IntelliJ for coursework.</body></html>
 ui.courseProject.form.settingsOptOut=Leave IntelliJ settings unchanged.
 ui.courseProject.form.currentSettingsText=<html><body>IntelliJ IDEA settings are already imported for <b>{0}</b>.</body><html>
-ui.courseProject.form.warningText=<html>(<b>Not recommended</b>. Only pick this option if you are sure you know what you are doing.)</html>
-ui.courseProject.view=Turn Project Into A+ Course Project
+ui.courseProject.form.warningText=<html>(Leaving the settings unchanged <b>not recommended</b>. Select this box only if you know what you are doing.)</html>
```

Comment:
Was this intended to be `Leaving the settings unchanged _is_...`?

Reply:
Ah, yes.